projects/nokogiri/project.yaml

```diff
@@ -7,9 +7,7 @@ auto_ccs:
   - "nokogiri-oss-fuzz@googlegroups.com"
 main_repo: "https://github.com/sparklemotion/nokogiri.git"
 sanitizers:
-  - address
```

Comment:
Just noticed you removed ASAN, is this intentional? I'd consider this likely the most important sanitizer

Reply:
@fuzzy-boiii23a